IbisLib/application.cpp

```diff
@@ -320,7 +320,10 @@ void Application::InitHardware()
     // Init hardware
     bool success = false;
     foreach( HardwareModule * module, m_hardwareModules )
+    {
```

Comment:
Normally, this shouldn't be needed as the Application::LoadPlugin method does that for all plugins (including Hardware modules). The only problem is that hardware is initialized before LoadPlugins is called. Can we simply call LoadPlugins earlier in the main function?